src/software/thunderscope/widget_setup_functions.py

```diff
@@ -135,6 +139,7 @@ def setup_gl_widget(
     gl_widget.add_layer(validation_layer)
     gl_widget.add_layer(trail_layer, False)
     gl_widget.add_layer(debug_shapes_layer, True)
+    gl_widget.add_layer(max_dribble_layer, False)
```

Comment:
this probably is useful when watching Thunderscope, so let's enable it by default